File: Uri/1036.py
from math import sqrt

x = raw_input()
x = x.split(' ')
a = float(x[0])
b = float(x[1])
c = float(x[2])
delta = b*b - 4*a*c


if(a == 0 or delta < 0):
    print "Impossivel calcular"
else:
    x1 = (-b + sqrt(delta))/(2*a)
    x2 = (-b - sqrt(delta))/(2*a)
    print "R1 = %.5f"%x1
    print "R2 = %.5f"%x2

File: Uri/1036.c
#include<stdio.h>
#include<stdlib.h>
#include<math.h>

int main(){
    float a, b, c, d, x1, x2;
    scanf("%f %f %f", &a, &b, &c);
    if(a == 0){
        printf("Impossivel calcular\n");
    }
    else{
        d = (b*b)-(4*a*c);
        if(d < 0){
            printf("Impossivel calcular\n");
        }
        else{
            x1 = (-b + (sqrt(d)))/(2*a);
            x2 = (-b - (sqrt(d)))/(2*a);
            printf("R1 = %.5f\n", x1);
            printf("R2 = %.5f\n", x2);
        }
    }
    return 0;
}
